.chloggen/1439-fix-HPA-updates.yaml

```diff
@@ -0,0 +1,17 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. operator, target allocator, github action)
+component: Autoscaler
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: This PR fixes issue where HPA fails to update when an additional metric is added to the spec.
```

Comment:
`Fix the issue where HPA fails to update when an additional metric is added to the spec.`

Reply:
fixed!